src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -434,6 +455,10 @@ h2::before {
   }
 }
 
+.capitalize {
+  text-transform: uppercase;
```

Comment:
Use the existing Vuetify helper class instead.

https://vuetifyjs.com/en/styles/text-and-typography/#transform